transaction/std/src/lib.rs

```diff
@@ -14,6 +14,9 @@ mod memo;
 mod memo_builder;
 mod transaction_builder;
 
+#[cfg(any(test, feature = "test-only"))]
```

Comment:
Optional/Equivalent: Name the feature `test_utils`

Reply:
this feature already exists in the `mc-transaction-std` stuff, it is used for the alternate sorting order for testing that validation of output sorting is working, so i just piggy backed on that